graphql-server/pkg/queries/validator.go

```diff
@@ -66,6 +76,17 @@ func (q *ValidatorQuery) NewQuery(model interface{}, includeAddresses []string)
 		})
 	}
 
+	if q.scopeValidatorStatus != "" {
+		statusQuery := q.session.NewSelect().Model((*models.ValidatorStatus)(nil)).ColumnExpr("validator_address")
+		switch q.scopeValidatorStatus {
+		case models.ValidatorStatusFilterActive:
+			statusQuery = statusQuery.Where("status = ?", stakingtypes.Bonded).Where("jailed IS ?", false)
+		case models.ValidatorStatusFilterInactive:
+			statusQuery = statusQuery.Where("jailed IS ?", true).WhereOr("status = ?", stakingtypes.Unbonding).WhereOr("status = ?", stakingtypes.Unbonded)
```

Comment:
not familiar with bun, but is where group required for this? or does chaining .WhereOr already do what we want? `... AND (... OR ... OR ...) AND ...`

Reply:
Both works, https://bun.uptrace.dev/guide/query-where.html#grouping